assets/sass/application.sass

```diff
@@ -5,6 +5,7 @@ $path: "/assets/images/"
 
 @import 'govuk/all'
 @import 'moj/all'
+@import "node_modules/@ministryofjustice/hmpps-digital-prison-reporting-frontend/dpr/all"
```

Comment:
should be able to import this via `dpr/all`, maybe the build css script needs updating?